internal/templatefn/fn.go

```diff
@@ -375,3 +377,7 @@ func concatOr(app page.AppData, list []string) string {
 func concatComma(list []string) string {
 	return strings.Join(list, ", ")
 }
+
+func penceToPounds(pence int) string {
+	return humanize.CommafWithDigits(float64(pence)/100, 2)
```

Comment:
I was going a bit crazy trying to roll my own version of this but happy for any alternative on the PR 👍

Reply:
I'm fine with this, but hope we'll never need commas

Reply:
Things will have gone very wrong on the affordability front if so